configuration/uart.md

```diff
@@ -21,31 +17,33 @@ Broadcom gives each UART a name and number in their SoC documentation, and this
-
-There are chips called ['level shifters'](https://en.wikipedia.org/wiki/Level_shifter) which convert voltages between levels. You can find them either as individual ICs for using in your own designs, or integrated into other devices to make them compatible. They're inexpensive and readily available from third parties.
+All the onboard Raspberry Pi UARTs are 3.3V ([CMOS](https://en.wikipedia.org/wiki/CMOS) [logic level](https://en.wikipedia.org/wiki/Logic_level)). Do not connect Raspberry Pi UARTs to 5V ([TTL logic level](https://en.wikipedia.org/wiki/Transistor%E2%80%93transistor_logic)) circuits, because the Raspberry Pi will be damaged. [Level shifters](https://en.wikipedia.org/wiki/Level_shifter) are inexpensive and readily available.
 
 ## Software support
 
-Raspberry Pi OS has built-in drivers for the UARTs built into the SoC, and common external types as well.
+Raspberry Pi OS ships with drivers for the onboard UARTs.
 
-The PL011 devices will appear as [character devices](https://en.wikipedia.org/wiki/Device_file#Character_devices) with names like `/dev/ttyAMA*`. If there are multiple PL011s, they're in the same _order_ as the Broadcom documentation, but not necessarily the same _number_. The mini UART always appears as character device named `/dev/ttyS0`, and there is never more than one. If you want to be specific about which hardware type you're using, use the `/dev/ttyAMA*` and `/dev/ttyS0` names.
+PL011s appear as [character devices](https://en.wikipedia.org/wiki/Device_file#Character_devices) `/dev/ttyAMA*`. If there are multiple PL011s, they're in the same _order_ as the Broadcom documentation, but not necessarily the same _number_. The mini UART always appears as `/dev/ttyS0`. If you want to be specific about hardware types, use the `/dev/ttyAMA*` and `/dev/ttyS0` names.
 
-There are also be symbolic links with names like `/dev/serial*`. If a UART is enabled and configured to appear on pins 8 and 10 of the GPIO header, `/dev/serial0` will link to that UART. If a UART is enabled and configured to connect to the onboard Bluetooth module, `/dev/serial1` links to that one. If you want to be specific about where a UART is connected, use the `/dev/serial*` names.
+There are also be symbolic links: `/dev/serial*`. If a UART is enabled and connected to pins 8 and 10 of the GPIO header, `/dev/serial0` will link to that UART. If a UART is enabled and connected to the onboard Bluetooth module, `/dev/serial1` links to that one. If you want to be specific about a UART connection, use the `/dev/serial*` names.
```

Comment:
"also be" -> "also"

Perhaps "connected" -> "internally connected"? (to differentiate it from anything a user might externally connect to the GPIO header)

Perhaps "link to" -> "point to"? :shrug: 

I'm not sure if the "If you want to be specific..." sentences (in both this and the previous paragraph) are useful - surely a user has to know where their UART is physically connected to, and so the `/dev/serial*` symlinks should always be preferred?

Reply:
> "also be" -> "also"

Yes

> Perhaps "connected" -> "internally connected"? (to differentiate it from anything a user might externally connect to the GPIO header)

Sure.

> I'm not sure if the "If you want to be specific..." sentences (in both this and the previous paragraph) are useful - surely a user has to know where their UART is physically connected to, and so the /dev/serial* symlinks should always be preferred?

I think finding the right name is really important, and I think you're right that we should point readers to `/dev/serial*`. My intention was to explain the relationship between the names and the hardware. The next commit will include a rewrite which puts `/dev/serial*`.